cluster-up/ssh.sh

```diff
@@ -40,7 +40,7 @@ if [[ $provider_prefix =~ okd.* ]]; then
         exit 1
     fi
     shift
-    ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -q -lcore -p $port core@127.0.0.1 -i ${ssh_key} $@
+    ssh -o UserKnownHostsFile=/dev/null -oStrictHostKeyChecking=no -q -lcore -p $port core@127.0.0.1 -i ${ssh_key} $@
```

Comment:
I have an unpublished PR that uses the latest kubevirtci, and the ssh.sh in there already adds the StrictHostKeyChecking to it. Maybe just wait for that?

Reply:
Let's see how this run goes had to fix `ssh.sh`